tasks.py

```diff
@@ -323,14 +349,22 @@ def start_controller(c, detached=False):
 def deploy(
     c,
     install_dep=True,
-    start_all=False,
+    start_all=True,
     start_controller_=False,
     start_nodes=False,
     celery_log_level=None,
     monetdb_image=None,
 ):
-    """(Re)Deploy everything.
-    The nodes will be deployed using the existing node config files."""
+    """
+    Install dependencies, (re)create all the containers and (re)start all the services.
```

Comment:
please elaborate more as to what "all the containers" and "all the services" means